### DISCUSSION TITLE
Example Application

### DISCUSSION THREAD

**Discussion by bowb** [hettie-d/NORM#6] [General]
I noticed mention of a java application (I don't remember where) that uses this framework. Is this available to examine? There was also a [reddit post](https://www.reddit.com/r/PostgreSQL/comments/m27nnp/contributing_to_the_no_object_relational_mapping/) last year that mentioned an application being developed.  I have reached out to the op of the post but haven't heard back yet. @tomjohnkin (This appears to be the same person)

It would be nice to see an example of an application that handles more complicated schemas, large datasets, and deep child relationships.

**Comment by hettie-d** [comment]
Hi! Thank you for your interest in NORM! The application which you mention is an actual working application that is used at Braviant Holdings, my place of employment in 2016-2021. The code of this application is not available because it is proprietary, but we have an example of Java code that calls the functions from this repository. At the moment, it is only available in the PostgreSQL Query Tuning Book, but I will try to get it into this repo.
